api.go

```diff
@@ -461,6 +475,7 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 		configurationsCh:      make(chan *configurationsFuture, 8),
 		bootstrapCh:           make(chan *bootstrapFuture),
 		observers:             make(map[uint64]*Observer),
+		leadershipTransferCh:  make(chan *leadershipTransferFuture, 64),
```

Comment:
I randomly picked that number. That was before I returned `ErrLeadershipTransferInProgress` instead of waiting to be able to write to the channel. I changed it to use `0` capacity.

Reply:
oh well, of course it needs to be 1. 🤦‍♂️